api/Validations/CPF/CPFAttributePattern.cs

```diff
@@ -0,0 +1,20 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace api.Validations;
+
+public class CPFAttributePattern : ValidationAttribute
+{
+    public CPFAttributePattern()
+        {
+        }
+
+        protected override ValidationResult? IsValid(object? value, ValidationContext validationContext)
+        {
+            var cardLengthInvalid = ((string)value).Length != 11;
```

Comment:
Se o tipo do objeto `value` não for string poderá lançar uma exceção. Nesse caso voce pode adicionar uma validação para verificar se o valor fornecido é do tipo string